planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -137,3 +159,54 @@ function validatePlaceDuration(duration) {
     }
     return true;
 }
+
+async function submitPlaceCallback(place, status) {
+    if (status == google.maps.places.PlacesServiceStatus.OK) {
+        let openingTime = 0;
+        let closingTime = 0;
+        if (place.opening_hours) {
+            // for MVP purposes, use Monday's hours. 
+            const openHours = place.opening_hours.periods[1].open.hours;
+            const openMinutes = place.opening_hours.periods[1].open.minutes;
+            const closeHours = place.opening_hours.periods[1].close.hours;
+            const closeMinutes = place.opening_hours.periods[1].close.minutes;
+            openingTime = TimeRange.getTimeInMinutes(openHours, openMinutes);
+            closingTime = TimeRange.getTimeInMinutes(closeHours, closeMinutes);
+        } else { 
+            // Set the event's opening hours to 8am - 5pm otherwise. 
+            openingTime = TimeRange.getTimeInMinutes(8, 0);
```

Comment:
Is there some way to inform users that these opening hours might not be accurate since the Place API doesn't have them?

Reply:
I can just display it on the card :D